easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -388,6 +388,8 @@ def configure_step(self, srcdir=None, builddir=None):
                 self.cfg['preconfigopts'],
                 DEFAULT_CONFIGURE_CMD,
                 generator,
+                '-Wno-deprecated',  # Suppress warnings about e.g. using OLD policies
+                '-Wno-dev',         # Suppress warnings targeted at developers of CMakeLists.txt
```

Comment:
Shouldn't we make sure that `cmake` is actually still used as configure command before doing this?

Reply:
Isn't that always the case as this uses `DEFAULT_CONFIGURE_CMD`